packages/nextjs/app/builders/0xeA2f0BB0144fbff64E6f6F492B999674f9393882/page.tsx

```diff
@@ -0,0 +1,121 @@
+"use client";
```

Comment:
It looks like you forgot to create a new branch for the logo issue, this is including your profile page.

Reply:
or you created the branch from you profile branch.

Reply:
Exactly! You should always fork from a synced `main`

Reply:
Thank for the feedback @carletex @BillSchumacher! I'm correcting this right now.